terraform/modules/len/lambdas/psycopg2/__init__.py

```diff
@@ -0,0 +1,134 @@
+"""A Python driver for PostgreSQL
```

Comment:
Unless I've missed it we don't have anything to zip this up do we?

Reply:
think it should be done correctly now